code/modules/factory/parts.dm

```diff
@@ -415,17 +415,17 @@ GLOBAL_LIST_INIT(module, list(
 	))
 
 /obj/item/factory_part/module_valk
-	name = "Valkyrie Automedical Armor System"
-	desc = "An unfinished Valkyrie Automedical Armor System module"
+	name = "Valkyrie automedical armor system"
```

Comment:
improper or lowercase

game only cares about the first letter of names rest doesnt matter

Reply:
Done